src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    static final FigType type = FigType.IsoscelesTrapezoid;
```

Comment:
redundant, in the draw method you can call `this.getClass().getSimpleName()` instead